multiqc/modules/gatk/gatk.py

```diff
@@ -25,25 +26,72 @@ def __init__(self):
 
         # Initialise the parent object
         super(MultiqcModule, self).__init__(
-            name='GATK',  anchor='gatk', target='GATK',
+            name='GATK', anchor='gatk', target='GATK',
             href='https://www.broadinstitute.org/gatk/',
             info=(" is a toolkit offering a wide variety of tools with a "
                   "primary focus on variant discovery and genotyping."))
 
         # Set up class objects to hold parsed data
         self.general_stats_headers = OrderedDict()
         self.general_stats_data = dict()
-        n = dict()
 
         # Call submodule functions
-        n['varianteval'] = self.parse_gatk_varianteval()
-        if n['varianteval'] > 0:
-            log.info("Found {} VariantEval reports".format(n['varianteval']))
+        n_reports_found = 0
+        n_reports_found += self.parse_gatk_base_recalibrator()
+        n_reports_found += self.parse_gatk_varianteval()
 
         # Exit if we didn't find anything
-        if sum(n.values()) == 0:
+        if n_reports_found == 0:
             log.debug("Could not find any reports in {}".format(config.analysis_dir))
             raise UserWarning
 
         # Add to the General Stats table (has to be called once per MultiQC module)
         self.general_stats_addcols(self.general_stats_data, self.general_stats_headers)
+
+    def parse_report(self, f, table_names):
+        """ Parse a GATK report https://software.broadinstitute.org/gatk/documentation/article.php?id=1244
+
+        Only GATTable entries are parsed.  Tables are returned as a dict of tables.
+        Each table is a dict of arrays, where names correspond to column names, and arrays
+        correspond to column values.
+
+        Args:
+            f (file handle): a file handle to a GATK report.
+            table_names (dict): a dict with keys that are GATK report table names
+                (e.g. "#:GATKTable:Quantized:Quality quantization map"), and values that are the
+                keys in the returned dict.
+
+        Returns:
+            {
+                table_1:
+                    {
+                        col_1: [ val_1, val_2, ... ]
+                        col_2: [ val_1, val_2, ... ]
+                        ...
+                    }
+                table_2:
+                    ...
+            }
+        """
+
+        data = dict()
+        while True:
```

Comment:
Ok, my bad for untested code suggestions :)

Why not just this?
```python
for line in f:
    line = line.rstrip()
    # ..etc
```

If you want the whole file (as a string rather than a list) then remove the `filehandles=True` from the `find_log_files` function